case class CustomException(errorMsg:String)  extends Exception(error:String)

 catch {
      case s: Exception => throw CustomException("This is a custom message")
    }

super(errorMessage)

